src/libraries/Directory.Build.targets

```diff
@@ -130,7 +130,8 @@
       <_GenAPICmd>$(_GenAPICmd) "@(IntermediateAssembly)"</_GenAPICmd>
       <_GenAPICmd>$(_GenAPICmd) --lib-path "$(RefPath.Trim('\/'))"</_GenAPICmd>
       <_GenAPICmd>$(_GenAPICmd) --out "$(_RefSourceFileOutputPath)"</_GenAPICmd>
-      <_GenAPICmd>$(_GenAPICmd) --exclude-attributes-list "$(_ExcludeAPIList)"</_GenAPICmd>
```

Comment:
@safern I'm wondering whether this is a bug. The file is named `_ExcludeAPIList` but it's being passed to the `--exclude-attributes-list` argument.

Reply:
No, this is correct. I think the naming was just messed up.